src/constants.js

```diff
@@ -0,0 +1,2 @@
+export const one_second = 1,
```

Comment:
please use all uppercase words for const names: [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage)